### DISCUSSION TITLE
Is there a way to not import matrix blocks that are empty in an entries json feed?

### DISCUSSION THREAD

**Discussion by houtaiwong-toast** [engram-design/FeedMe#1079] [Q&A]
I was wondering if there is a way to not import/create empty matrixBlocks from a feed of entries. I have a channel I'm trying to migrate a bunch of entries that share the same content builder to a new site, but many of these entries do not use some of the matrix blocks that the content builder offers. 

For example, if I had a matrix of "body, heading, category", some entries may have just the "body" and "heading" block but not category.

So in my json feed, I've basically nullified all the empty block values, but when importing it just creates those empty blocks with no values. 

Is there a way to do that?

**Comment by amphibian** [comment]
Recently ran into this as well. It's an issue because you need to have all blocks defined in your JSON in order to map them to your Matrix field, but you'll often have no content in those blocks.
